src/components/LessonInfoDisplay/LessonInfoDisplay.js

```diff
@@ -16,27 +31,33 @@ function LessonInfoDisplay(props) {
         <div className="Column">
           <div className="LessonDataTitle">WORDS</div>
           <div className="LessonData">
-            {props.words.map(word => (
-              <div>{word}</div>
+            {props.words.map((word, index) => (
+              <div key={index}>{word}</div>
             ))}
           </div>
         </div>
         <div className="Column">
           <div className="LessonDataTitle">STUDENTS</div>
           <div className="LessonData">
-            {props.studentNames.map(name => (
-              <div>{name}</div>
+            {props.studentNames.map((name, index) => (
+              <div key={index}>{name}</div>
             ))}
           </div>
         </div>
       </div>
-      <div>
-        <button onClick={() => handleClose()} className="CloseButton">
+      <div className="ButtonContainer">
+        <button onClick={() => handleClose()} className="LessonButton">
```

Comment:
you can just do `onClick = {handleClose}` here, also use capital B Button from react-bootstrap

Reply:
changing it to {handleClose} rather than {() => handleClose()} will actually call the function immediately on render of the component rather than only when the button isClicked

Reply:
oh really nvm then

Reply:
wait i take that back notice that you don't put the parentheses in what i gave you

Reply:
wait wdym